src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/CanonicalXmlNodeList.cs

```diff
@@ -43,24 +43,24 @@ public void Clear()
             _nodeArray.Clear();
         }
 
-        public bool Contains(object value)
+        public bool Contains(object? value)
         {
             return _nodeArray.Contains(value);
         }
 
-        public int IndexOf(object value)
+        public int IndexOf(object? value)
         {
             return _nodeArray.IndexOf(value);
         }
 
-        public void Insert(int index, object value)
+        public void Insert(int index, object? value)
```

Comment:
I assume the `?` is here because of the interface implementation? the getters can still be non-null though because that doesn't break interface

Reply:
Yes, it's to satisfy the interface. `Item` is already non-null.